app/views/gardens/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1><%= @garden.name %></h1>
+<p>Plants: <% @garden.plants.each do |plant| %></p>
+  <% if plant.days_to_harvest < 100 %>
+    <p><%= plant.name %></p>
+  <% end %>
+<% end %>
```

Comment:
Instead of using ruby here to iterate through the plants and only print out the ones whose harvest days are less than 100, it would be better to create a Garden model method that returns a distinct list of plants from that garden that has less than 100 days to harvest.